src/components/Pagination.js

```diff
@@ -0,0 +1,25 @@
+import React, { useState } from 'react';
+/* eslint-disable react/prop-types */
```

Comment:
- [ OPTIONAL] Kindly remove the eslint disable line and instead fix the issue in the code.